plugins/mgm/src/main/kotlin/net/corda/cli/plugins/mgm/GenerateGroupPolicy.kt

```diff
@@ -63,109 +64,76 @@ class GenerateGroupPolicy(private val output: GroupPolicyOutput = ConsoleGroupPo
-                "ecdhKey" to "-----BEGIN PUBLIC KEY-----\nMCowBQYDH8Tc=\n-----END PUBLIC KEY-----\n",
-                "keys" to listOf(
-                    "-----BEGIN PUBLIC KEY-----\nMFkwEwYHgcIALw==\n-----END PUBLIC KEY-----\n"
+            "p2pParameters" to mapOf(
+                "sessionTrustRoots" to listOf(
+                    "-----BEGIN CERTIFICATE-----\n{truncated for readability}\n-----END CERTIFICATE-----",
+                    "-----BEGIN CERTIFICATE-----\n{truncated for readability}\n-----END CERTIFICATE-----",
                 ),
-                "endpoints" to listOf(
-                    mapOf(
-                        "url" to "https://mgm.corda5.r3.com:10000",
-                        "protocolVersion" to 1
-                    ),
-                    mapOf(
-                        "url" to "https://mgm-dr.corda5.r3.com:10000",
-                        "protocolVersion" to 1
-                    )
+                "tlsTrustRoots" to listOf(
+                    "-----BEGIN CERTIFICATE-----\n{truncated for readability}\n-----END CERTIFICATE-----",
+                    "-----BEGIN CERTIFICATE-----\n{truncated for readability}\n-----END CERTIFICATE-----",
+                    "-----BEGIN CERTIFICATE-----\n{truncated for readability}\n-----END CERTIFICATE-----",
                 ),
-                "platformVersion" to 1,
-                "softwareVersion" to "5.0.0",
-                "serial" to 1
+                "sessionPki" to "Standard",
+                "tlsPki" to "Standard",
+                "tlsVersion" to "1.3",
+                "protocolMode" to "Authentication_Encryption",
             ),
-            "identityPKI" to "Standard",
-            "identityKeyPolicy" to "Combined",
             "cipherSuite" to mapOf(
                 "corda.provider" to "default",
                 "corda.signature.provider" to "default",
                 "corda.signature.default" to "ECDSA_SECP256K1_SHA256",
                 "corda.signature.FRESH_KEYS" to "ECDSA_SECP256K1_SHA256",
                 "corda.digest.default" to "SHA256",
-                "corda.cryptoservice.provider" to "default"
+                "corda.cryptoservice.provider" to "default",
             ),
-            "roles" to mapOf(
-                "default" to mapOf(
-                    "validator" to "net.corda.v5.mgm.DefaultMemberInfoValidator",
-                    "requiredMemberInfo" to emptyList<Any>(),
-                    "optionalMemberInfo" to emptyList<Any>()
-                ),
-                "notary" to mapOf(
-                    "validator" to "net.corda.v5.mgm.NotaryMemberInfoValidator",
-                    "requiredMemberInfo" to listOf("notaryServiceParty"),
-                    "optionalMemberInfo" to emptyList<Any>()
-                )
+        )
+    }
+
+    private val defaultMembers by lazy {
+        listOf(
+            mapOf(
+                "name" to "C=GB, L=London, O=Alice",
+                "keyAlias" to "alice-alias",
```

Comment:
I think `keyAlias` and `rotatedKeyAlias` values can be removed. We used those for deterministic key gen but the static mode doesn't generate keys deterministically anymore so they aren't required. 
These can be removed elsewhere in this class also (e.g. `membersFromFile`).
The design doc is out of date for that unfortunately.